src/components/InteractiveQuestionView.tsx

```diff
@@ -77,7 +77,7 @@ export const InteractiveQuestionView = ({ isSaveEnabled = false }: Props) => {
       </Flex>
 
       {view === "viz" && (
-        <Box h="500px">
+        <Box h={500}>
```

Comment:
Just wanna confirm. This seems to change nothing as the new value will basically result in the same pixel value.

Reply:
yep, I'm not sure why I changed that to {500}, let me revert that